src/shared/components/error-screen/url-errors/MissingFeatureError.tsx

```diff
@@ -0,0 +1,62 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+
+import { getDisplayName } from 'src/shared/components/selected-species/selectedSpeciesHelpers';
+
+import AlertButton from 'src/shared/components/alert-button/AlertButton';
+import { PrimaryButton } from 'src/shared/components/button/Button';
+
+import styles from './UrlError.scss';
+
+type GenomeFields = {
+  common_name: string | null;
+  scientific_name: string;
+};
+
+type Props = {
+  featureId: string;
+  genome: GenomeFields | null;
+  showTopBar?: boolean;
+  onContinue: () => void;
+};
+
+const MissingFeatureError = (props: Props) => {
+  const { featureId, genome, showTopBar, onContinue } = props;
+
+  const speciesDisplayName = genome ? getDisplayName(genome) : 'this species';
+
+  return (
+    <div>
+      {showTopBar && <div className={styles.topBar} />}
+      <main className={styles.main}>
+        <AlertButton className={styles.alertButton} />
+        <div className={styles.errorMessage}>
+          We do not recognize "{featureId}" in {speciesDisplayName}
```

Comment:
Was `"{featureId}"` meant to be in quotes?

Reply:
Yes. Here's the [XD](https://xd.adobe.com/view/b6d9a69d-468f-44b1-9333-419e228aa5ff-e38a/screen/c9a7ae6e-2e55-4991-8196-11a664b27424); and I used double quotes instead of single quotes.

[Link to an example on the site](http://missing-genomes.review.ensembl.org/entity-viewer/iwgsc/TraesCS3D02G273600?view=transcripts)